modules/snpsift/main.nf

```diff
@@ -0,0 +1,40 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process SNPSIFT {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::snpsift=4.3.1t" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/snpsift:4.3.1t--hdfd78af_3"
+    } else {
+        container "quay.io/biocontainers/snpsift:4.3.1t--hdfd78af_3"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: split_vcfs
+    path "versions.yml"          , emit: versions
+
+    script:
+    def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    SnpSift \\
+        split \\
```

Comment:
If you use a specific subcommand, the module should probably be in snpsift/sift and called `SNPSIFT_SIFT` (see the samtools modules for example) That way more snpsift subcommand modules can be added in the same way